app/models/wpcc/period.rb

```diff
@@ -3,7 +3,7 @@ class Period
     include ActiveModel::Model
     include WpccConfig
 
-    LEGAL_MINIMUM_CONTRIBUTION_PERCENT = 1
+    LEGAL_MINIMUM_CONTRIBUTION_PERCENT ||= 1
```

Comment:
:question: Isn't the whole point of a constant to only get assigned once? So you get the memoization behaviour by default, right?

Reply:
Good point, I got into the habit when using constants in modules to avoid/silence warnings when the objects got reloaded. Similarly to [here](https://stackoverflow.com/questions/7953611/solutions-to-the-annoying-warning-already-initialized-constant-message)

In this case it makes no sense 👍

Reply:
oh had no idea about this. Good to know!